Tests/SpelledPitchTests/PitchSpellerTests.swift

```diff
@@ -13,12 +13,12 @@ import Pitch
 class PitchSpellerTests: XCTestCase {
 
     // MARK: - AssignedNode
-
-    func testAssignedNodeComparable() {
-        let a = PitchSpeller.AssignedNode(Cross(0,.down), .down)
-        let b = PitchSpeller.AssignedNode(Cross(1,.up), .down)
-        XCTAssertLessThan(a,b)
-    }
+//
```

Comment:
Should this be replaced with `FlowNode` comparison or sim.?

Reply:
Not sure that this comparable conformance is needed in the context anymore, because we are no longer sorting `Node`s by value. Thoughts?

Reply:
Ahh, then all good!